docs/guide/custom-functions.md

```diff
@@ -140,16 +220,83 @@ the calculations.
 // arguments here are displayed just to show the structure
 public hyper(ast, state) {
     return 'Hyperformula'.length;
+  };
+```
+
+### `runFunction()`
+
+To test your function's configuration, run the built-in `runFunction()` function.
```

Comment:
`runFunction` will validate arguments according to configuration but from this sentence and example below it looks like it's the only purpose. As the name indicates, it need to run the custom function. So it wraps your calculations, prepares the arguments and then parses results. 

But I'm not sure if I have all the details right, CC @izulin

Reply:
@wojciechczerniak - updated: 1d1bada

Reply:
Yes, `runFunction` does all the heavy lifting of argument sanitization, output sanitization, vectorization, flag handling etc.